shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/core/execute/PipelineJobWorker.java

```diff
@@ -41,7 +41,7 @@ public static void initialize() {
                 return;
             }
             log.info("start worker initialization");
-            new PipelineJobExecutor().start();
+            PipelineJobExecutor.registerListener();
```

Comment:
`PipelineJobExecutor.getInstance();` could be called here, it doesn't matter if it's missed